interfaces/python/helicsPython.i

```diff
@@ -1,3 +1,5 @@
+%feature("doxygen:ignore:forcpponly", range="end");
```

Comment:
do we need to add this command in the python2 .i file as well?

Reply:
Good catch! Resolved in a8794035029c1b426e9161e7cad4d6b57bd0f2d0.